devices/homeetec.js

```diff
@@ -0,0 +1,360 @@
+            
+        } else if ([DP_KEYS.Backlight, DP_KEYS.ChildLock, DP_KEYS.Switch, DP_KEYS.Switch1, DP_KEYS.Switch2].includes(key)) {
+            result[key] = getOnState(value);
+            meta.logger.debug(`homeetec_cover (FZ) DEBUG boolean, dp: ${dp}, key: ${key}, value: ${value}, dpValue: ${JSON.stringify(dpValue)}, result: ${JSON.stringify(result)}`);
+        }
+        
+        if (Object.keys(result).length > 0) {
+            return result;
+        }
+    },
+};
+
+const getDefintions = (model = null) => {
+    const definitions = [
+        {
+            fingerprint: [{
+                modelID: 'TS0001',
+                manufacturerName: '_TZ3000_bmqxalil',
+            }],
+            model: 'TS0001_switch_1_gang',
+            vendor: 'TuYa',
+            description: '1-Gang switch with backlight',
+            fromZigbee: [fz.on_off, tuya.fz.power_on_behavior_2, tuya.fz.backlight_mode_off_on],
+            toZigbee: [tz.on_off, tuya.tz.power_on_behavior_2, tuya.tz.backlight_indicator_mode_2],
+            exposes: [
+                e.switch(),
+                e.power_on_behavior(),
+                exposes.binary('backlight_mode', ea.STATE_SET, 'on', 'off').withDescription('Backlight mode'),
+            ],
+            configure: async (device, coordinatorEndpoint, logger) => {
+                await tuya.configureMagicPacket(device, coordinatorEndpoint, logger);
+                await reporting.bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
+            },
+        },
+        {
+            fingerprint: [{
+                modelID: 'TS0002',
+                manufacturerName: '_TZ3000_in5qxhtt',
+            }],
+            model: 'TS0002_switch_2_gang',
+            vendor: 'TuYa',
+            description: '2-Gang switch with backlight',
+            fromZigbee: [fz.on_off, tuya.fz.power_on_behavior_2, tuya.fz.backlight_mode_off_on],
+            toZigbee: [tz.on_off, tuya.tz.power_on_behavior_2, tuya.tz.backlight_indicator_mode_2],
+            exposes: [
+                e.switch().withEndpoint('l1'),
+                e.switch().withEndpoint('l2'),
+                e.power_on_behavior(),
+                exposes.binary('backlight_mode', ea.STATE_SET, 'on', 'off').withDescription('Backlight mode'),
+            ],
+            endpoint: (device) => {
+                return {'l1': 1, 'l2': 2};
+            },
+            meta: {multiEndpoint: true, disableDefaultResponse: false},
+            configure: async (device, coordinatorEndpoint, logger) => {
+                await tuya.configureMagicPacket(device, coordinatorEndpoint, logger);
+                await reporting.bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
+                await reporting.bind(device.getEndpoint(2), coordinatorEndpoint, ['genOnOff']);
+            },
+        },
+        {
+            fingerprint: [{
```

Comment:
Could you check if this device works with:

```js
{
    fingerprint: tuya.fingerprint('TS0003', ['_TZ3000_pv4puuxi']),
    model: 'TS0003_switch_3_gang',
    vendor: 'TuYa',
    description: '3 gang switch',
    extend: tuya.extend.switch({powerOnBehavior2: true, backlightModeOffOn: true, endpoints: ['l1', 'l2', 'l3']}),
    endpoint: (device) => {
        return {'l1': 1, 'l2': 2, 'l3': 3};
    },
    meta: {multiEndpoint: true, disableDefaultResponse: false},
    configure: async (device, coordinatorEndpoint, logger) => {
        await tuya.configureMagicPacket(device, coordinatorEndpoint, logger);
        await reporting.bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
        await reporting.bind(device.getEndpoint(2), coordinatorEndpoint, ['genOnOff']);
        await reporting.bind(device.getEndpoint(3), coordinatorEndpoint, ['genOnOff']);
    },
}
```

Reply:
I used left, right and center as it was in the other examples and match to the order of the physical buttons

```javascript
fingerprint: [{
            modelID: 'TS0003',
            manufacturerName: '_TZ3000_pv4puuxi',
        }],
        model: 'TS0003_switch_3_gang',
        vendor: 'TuYa',
        description: '3-Gang switch with backlight',
        extend: tuya.extend.switch({powerOnBehavior2: true, backlightModeOffOn: true, endpoints: ['left', 'center', 'right']}),
        endpoint: (device) => {
            return {'left': 1, 'center': 2, 'right': 3};
        },
        meta: {multiEndpoint: true},
        configure: async (device, coordinatorEndpoint, logger) => {
            await tuya.configureMagicPacket(device, coordinatorEndpoint, logger);
            await reporting.bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
            await reporting.bind(device.getEndpoint(2), coordinatorEndpoint, ['genOnOff']);
            await reporting.bind(device.getEndpoint(3), coordinatorEndpoint, ['genOnOff']);
        },
```